lib/components/DropdownInputGroup.jsx

```diff
@@ -1,9 +1,9 @@
-import React, { useState } from 'react'
+import { Menu, MenuButton, MenuItem, MenuItems, MenuPopover } from '@reach/menu-button'
+import { positionMatchWidth } from '@reach/popover'
 import classnames from 'classnames'
 import FeatherIcon from 'feather-icons-react'
-import { Menu, MenuList, MenuButton, MenuItem, MenuPopover, MenuItems } from '@reach/menu-button'
-import { positionMatchWidth } from '@reach/popover'
-import { DEFAULT_INPUT_GROUP_CLASSES } from 'lib/constants'
+import { DEFAULT_INPUT_GROUP_CLASSES, DEFAULT_INPUT_LABEL_CLASSES } from 'lib/constants'
+import React, { useState } from 'react'
```

Comment:
😂 that import line just doesn't want to stay at the top

Reply:
Ah damn, I forgot to fix this pr too lol